api/advanced-metrics.yaml

```diff
@@ -0,0 +1,70 @@
+paths:
+  /advanced/config:
+    post:
+      tags: ['Advanced Metrics']
+      operationId: set_advanced_metrics
+      description: |-
+        An API that extends the base Traffic Generator configuration with advanced metric configuration.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '../advanced/config.yaml#/components/schemas/AdvancedMetrics.Config'
+      responses:
+        '200':
+          $ref: '../result/request.yaml#/components/responses/Success'
+        '409':
+          $ref: '../result/request.yaml#/components/responses/Conflict'
+        '590':
+          $ref: '../result/request.yaml#/components/responses/Fail'
```

Comment:
i think we mentioned at some point, one of these response codes will be the means an implementation detailed back to a client if some capabilities in the config is not supported.

is there any guidance on the format/response code implementation should adhere to?